src/ui/Facet/FacetSettings.ts

```diff
@@ -327,10 +330,11 @@ export class FacetSettings extends FacetSort {
     directionAscendingSection.appendChild(iconAscending);
     directionAscendingSection.appendChild(directionItemsAscending);
     new AccessibleButton()
-      .withElement(directionAscendingSection)
-      .withLabel(l('Ascending'))
-      .withSelectAction((e: Event) => this.handleDirectionClick(e, 'ascending'))
+      .withElement(ascending)
```

Comment:
Even though the `ascending` element is smaller, possibly due to the removed condition below with the `.parentElement.parentElement`, only clicking `ascending` worked.

This replacement is done since most attributes for this button are set on the `ascending` element already. it seemed preferable to have every accessibility attribute and event on the same element.